### PR TITLE
merge documentation update

### PR DESCRIPTION
**Pull request by dasturge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,136 +6,147 @@ to run the subject(s).
...
 
-You will need to load the image onto your docker service by running the following command:
+Before running, you will need to load the image onto your Docker service by running the following command:
 
 ```{bash}
 docker pull dcanlabs/abcd-hcp-pipeline
 ```
 
-It is common to receive a "no space left on device" error during this pull process. You may need to clean up any old/dangling images and containers from the docker registry, and possibly increase the amount of space allocated to Docker.
+If you receive a "no space left on device" error during this pull process.
```
Comment (kathy-snider) [inline README.md]: , instead of .

File: README.md [hunk 2]
```diff
@@ -6,136 +6,147 @@ to run the subject(s).
...
+You may need to clean up any old/dangling images and containers from the docker registry, and possibly increase the amount of space allocated to Docker.
```
Comment (kathy-snider) [inline README.md]: start with 'y' instead of 'Y'. Also might want to tell them they have to contact ACC for this.

File: README.md [hunk 3]
```diff
@@ -6,136 +6,147 @@ to run the subject(s).
...
-special pipeline options:
-  options which pertain to an alternative pipeline or an extra stage which is not
-   inferred from the bids data.
+Special pipeline options:
+  Options which pertain to an alternative pipeline or an extra stage which is not
+   inferred from the BIDS data.
 
-  --custom-clean JSON   runs dcan cleaning script after the pipeline
+  --custom-clean JSON   Runs DCAN cleaning script after the pipeline
                         completessuccessfully to delete pipeline outputs based
```
Comment (kathy-snider) [inline README.md]: Fix completesuccessfully.

File: app/run.py [hunk 4]
```diff
@@ -73,113 +73,115 @@ def generate_parser(parser=None):
...
-             'provide your own freesurfer license. The license can be '
+        help='If using docker or singularity, you will need to acquire and '
+             'provide your own FreeSurfer license. The license can be '
              'acquired by filling out this form: '
              'https://surfer.nmr.mgh.harvard.edu/registration.html'
     )
     parser.add_argument(
         '--all-sessions', dest='collect', action='store_true',
-        help='collapses all sessions into one when running a subject.'
+        help='Collapses all sessions into one when running a subject.'
```
Comment (kathy-snider) [inline app/run.py]: Where is the data required for this? Should there be 1 fmap dir or does every session need one, etc?

File: app/run.py [hunk 5]
```diff
@@ -73,113 +73,115 @@ def generate_parser(parser=None):
...
+             'frequency of approx. 1 Hz or more (TR<=1.0). Default is no '
+             'filter.'
     )
     extras = parser.add_argument_group(
-        'special pipeline options',
-        description='options which pertain to an alternative pipeline or an '
-                    'extra stage which is not\n inferred from the bids data.'
+        'Special pipeline options',
+        description='Options which pertain to an alternative pipeline or an '
+                    'extra stage which is not\n inferred from the BIDS data.'
```
Comment (kathy-snider) [inline app/run.py]: Do you mean to have the \n after not?

File: app/run.py [hunk 6]
```diff
@@ -73,113 +73,115 @@ def generate_parser(parser=None):
...
+        help='Print run commands for each stage to shell then exit.'
     )
     runopts.add_argument(
         '--ignore-expected-outputs', action='store_true',
-        help='continues pipeline even if some expected outputs are missing.'
+        help='Continues pipeline even if some expected outputs are missing.'
     )
 
-
     return parser
```
Comment (kathy-snider) [inline app/run.py]: I am at the end, but I don't know how to stop this!

File: app/run.py (hunk 5 above)
Comment (dasturge) [reply app/run.py]: Yes.  argparse "descriptions" are formatted raw, unlike the help text.  It was to get the citations at the end to format properly.

**Pull request merged**